src/components/form/field_search/field_search.js

```diff
@@ -31,13 +31,18 @@ const propTypes = {
    * when `true` creates a shorter height input
    */
   compressed: PropTypes.bool,
+  /**
+   * Shows a clear button when set to true
```

Comment:
Would be nice to be more descriptive here like:
```suggestion
   * Shows a button that quickly clears any input
```